src/Workspaces/CoreTestUtilities/Microsoft.CodeAnalysis.Workspaces.Test.Utilities.csproj

```diff
@@ -31,6 +31,9 @@
     <PackageReference Include="xunit.extensibility.execution" Version="$(xunitextensibilityexecutionVersion)" />
     <PackageReference Include="Xunit.Combinatorial" Version="$(XunitCombinatorialVersion)" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
```

Comment:
net472 has some transient dependency to `System.Diagnostics.DiagnosticSource` 7.0.2 so I add this.
(Not sure the dependency chain, because there is no projec.deps file for net472)
Given `System.Diagnostics.DiagnosticSource` support .netstandard 2.0 I feel it should be fine